Makefile

```diff
@@ -388,6 +397,11 @@ $(RELEASE_DIR_K8S_MANIFESTS):
 	    'mkdir -p $(RELEASE_DIR_K8S_MANIFESTS)/`dirname FILE`;\
 	    cp $(MANIFEST_SRC)/FILE $(RELEASE_DIR_K8S_MANIFESTS)/`dirname FILE`;'
 
+	# For older releases, also copy the "non-hosted" manifests to be consistent.
```

Comment:
Eh I'm still not sure if this will match the previous version. See https://github.com/projectcalico/calico/blame/a6516f5426152f937b6e19e214995f2db8e9a8fd/Makefile#L247

Previously the find tacked on `/hosted` to where it searched and then did this separate copy.

I guess you could have the MANIFEST_SRC (for old versions) include the `/hosted` and then in this section have `$(MANIFEST_SRC)/../*.yaml`.

Reply:
I think I'm wrong about my proposed solution.

Reply:
Yeah, this will result in:

- Manifests having an extra `hosted` in the directory path within the tar
- Including a few extra manifests (e.g., the app-layer-policy ones).

WDYT? Is that a problem? We don't guarantee that the contents of this tar file will be exactly the same. Though, I guess it would be nice to do so for patch releases at least.

Reply:
I'm not sure it will have an extra hosted. It looks like the previous included hosted in the path for the hosted ones, so that may not be an issue.
I don't know of a specific problem, I just noticed a difference and didn't want to be missing something or (my bigger concern) that we possibly had manifests with the same name and would be overwriting some.
If you've checked this out and things look similar to a previous one then I'm happy.

Reply:
Since you removed the `hosted` from the MANIFEST_SRC, is this additional copy even needed now? I think the find would pick up what this copy is grabbing.

Reply:
> that we possibly had manifests with the same name and would be overwriting some

I don't think this is the case, since we maintain the original directory structure so everything should be namespaced that way.

> Since you removed the hosted from the MANIFEST_SRC, is this additional copy even needed now? I think the find would pick up what this copy is grabbing.

 Good point - I removed and tried locally and the output looks the same.